exercises/concept/simple-calculator/.docs/hints.md

```diff
@@ -2,7 +2,6 @@
 
 ## General
 
-- Knowing about conditional statements such as `if/else` statements or `switch`
-is a prerequisite.
-- Having a list of permitted arguments can be useful and can make your code more
-readable.
+- Knowing about control flow such as `if/else` or `unless` is a prerequisite.
+- Having a collection of permitted arguments can be useful and can make your
```

Comment:
```suggestion
- Having a collection of permitted operations can be useful and can make your
```

Though they may be arguments, the idea is that they are the permitted operations.